arangod/GeneralServer/GeneralServerFeature.cpp

```diff
@@ -202,14 +208,23 @@ void GeneralServerFeature::collectOptions(
   options->addOldOption("server.keep-alive-timeout", "http.keep-alive-timeout");
   options->addOldOption("no-server", "server.rest-server");
 
+  options
+      ->addOption("--server.send-telemetrics",
+                  "Whether to enable the telemetrics API.",
```

Comment:
The name `send-telemetrics` is probably misleading. 
It is more about whether the server _has_ a telemetrics API or not.
Apart from that the switch is a bit problematic because you can enable the telemetrics API with this option, but then set `--server.support-info-api-policy` to `disabled`, which will effectively turn the API off again.